rpm/rpm.go

```diff
@@ -62,27 +58,24 @@ var osVendors = []string{
 	"SUSE",                  // SUSE Linux Enterprise
 	"openSUSE",              // openSUSE
 	"Microsoft Corporation", // CBL-Mariner
+	"Rocky",                 // Rocky Linux
 }
 
 type rpmPkgAnalyzer struct{}
 
+func newRPMPkgAnalyzer() *rpmPkgAnalyzer {
+	return &rpmPkgAnalyzer{}
+}
+
+type RPMDB interface {
```

Comment:
I don't remember which linter catches this, I think go vet, but it's not useful to start a type name with the same word as the package name, especially when dot imports are almost never used.

Reply:
All of this is code from upstream, I am not going to fix this.